test-runner/build.sbt

```diff
@@ -36,13 +36,17 @@ libraryDependencies ++= addSparkDependencies(sparkHome.value) ++ Seq(
    ExclusionRule(organization = "commons-beanutils"),
    ExclusionRule(organization = "org.apache.hadoop", name ="hadoop-yarn-api")),
  "org.scalatest"     %% "scalatest"       % "2.2.4",
- "com.typesafe"      %  "config"          % "1.2.1"
+ "com.typesafe"      %  "config"          % "1.2.1",
+ "com.amazonaws" % "aws-java-sdk" % "1.0.002",
+ "commons-io" % "commons-io" % "2.4"
```

Comment:
Please follow the same indentation. We don't have a strict style guide, but following the style around you is always a good approximation :)